tools/commit-checker/check.py

```diff
@@ -139,8 +153,12 @@ def get_single_jira_issue(issue_id, **kwargs):
     Returns a single ICUIssue for the given issue ID.
     """
     jira_url, jira = get_jira_instance(**kwargs)
-    jira_issue = jira.issue(issue_id)
-    print("Loaded single issue %s" % issue_id, file=sys.stderr)
+    try:
+        jira_issue = jira.issue(issue_id)
+    except JIRAError as e:
+        return None
+    if debug:
```

Comment:
I like this to be printed by default when running check.py for BRS because the script is slow and these messages indicate when there is progress.